lib/twirp/rails/rack/conditional_post.rb

```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 require "rack/conditional_get"
+begin
+  # Added in rack 2.2
+  require "rack/version"
+rescue LoadError
+  # Release version is in the "rack" file before moving to "rack/version"
+  require "rack"
```

Comment:
I'm torn on maybe just always doing the full rack require. It mostly just sets up a pile of autoload directives.

Reply:
hm, if you're using this you're probably requiring all of rack anyway. 🤷

Reply:
Actually, scrapping this nuance. We already have rails >= 7 which requires rack >= 2.2.0